lib/publishers/fetch.rb

```diff
@@ -25,6 +26,7 @@ def fetch(uri:, limit: 10, follow_all_redirects: false, follow_local_redirects:
 
         host = request["host"]
       end
+      response = SsrfFilter.get(uri, max_redirects: limit, request_proc: request_proc, http_options: {open_timeout: 8})
```

Comment:
needed?

Reply:
This is how we've always done it for our protection